Modules/_datetimemodule.c

```diff
@@ -2184,7 +2181,7 @@ delta_new(PyTypeObject *type, PyObject *args, PyObject *kw)
         "minutes", "hours", "weeks", NULL
     };
 
-    if (PyArg_ParseTupleAndKeywords(args, kw, "|OOOOOOO:__new__",
+    if (PyArg_ParseTupleAndKeywords(args, kw, "|OOOOOOO:timedelta.__new__",
```

Comment:
I agree. I changed it only because I changed other similar *_new functions in Modules/_datetimemodule.c, and this `:__new__` seemed like a mistake that I should correct while I am there.
(anyway, I will remove this change.)